api/store/mongo/migrations.go

```diff
@@ -17,7 +17,7 @@ import (
 func ApplyMigrations(db *mongo.Database) error {
 	logrus.Info("Creating lock for the resource migrations")
 
-	lockClient := lock.NewClient(db.Collection("locks", options.Collection().SetWriteConcern(writeconcern.New(writeconcern.WMajority()))))
+	lockClient := lock.NewClient(db.Collection("locks", options.Collection().SetWriteConcern(writeconcern.New(writeconcern.WMajority())))) //nolint:staticcheck
```

Comment:
Please fix this in a specific commit. Could this be fixes in another PR as this is unrelated.